src/components/record/record-action-buttons.tsx

```diff
@@ -0,0 +1,67 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_record.scss';
```

Comment:
I don't see the point of this component. It's not adding any value and you should just move this to be inside the `record.tsx`.